Documentation/Security/GuidelinesIntegrators/GlobalTypo3Options.rst.txt

```diff
@@ -54,26 +56,6 @@ which means "localhost" only.
 The PHP variable reads: :php:`$GLOBALS['TYPO3_CONF_VARS']['SYS']['devIPmask']``
 
 
-.. _security-global-typo3-options-enabledBeUserIPLock:
-
-enabledBeUserIPLock
-"""""""""""""""""""
-
-If this configuration is enabled (value `1`), backend user accounts
-can be locked to specific IP addresses by using user/group TSconfig.
-Possible values are: `0` or `1` (boolean), where `0` deactivates the
-option and `1` enables it (default).
-
```

Comment:
The sorting in a reference is usually to show the syntax with options first  and then the explanation.
I wouldn't use a sentence to display the syntax.